golem-worker-executor-base/src/workerctx.rs

```diff
@@ -73,6 +74,7 @@ pub trait WorkerCtx:
     + Send
     + Sync
     + Sized
+    + DurableWorkerCtxView<Self>
```

Comment:
This should not be necessary - and breaks the (now unused) purpose of this trait that it is independent of `DurableWorkerCtx` (that's just one implementation - originally we had a non-durable one too).

We already have `as_wasi_http_view` and `as_wasi_view` below, isn't that enough?

Reply:
Also, `Ctx` decides whether worker context handles things related to durability.  So this implies the above comment from @vigoo should be made work if it's not working now, in my view - which  is simply `.as_wasi_view` where ever it is required.

Reply:
adressed